utils/src/main/java/org/jclouds/karaf/utils/EnvHelper.java

```diff
@@ -238,4 +240,12 @@ private static String getValueOrPropertyOrEnvironmentVariable(
         }
         return value;
     }
+    
+   private static String getCredentialValue(String provider, String credential) {
+      return credential != null && isGoogleCloud(provider) ? getGoogleCredentialFromJsonFileIfPath(credential) : credential;
+   }
+   
+   private static boolean isGoogleCloud(String provider) {
+      return provider != null && provider.startsWith("google");
```

Comment:
> Yes, it can be null if the user hasn't provided explicitly the parameter or the environment variable.

Ah, so we are not checking for this beforehand? In that case, clear.

Thanks for clarifying!